docs/source/start.ipynb

```diff
@@ -1,13 +1,15 @@
 {
  "cells": [
   {
+   "attachments": {},
```

Comment:
Nit: can you back out all these extra args 😅

Reply:
These additional cell attachment tags are going to be the end of me 😭. I'm opened [a new issue](https://github.com/alteryx/evalml/issues/4222) to get this added to our linting tools.